win32ss/user/ntuser/winpos.c

```diff
@@ -2711,10 +2711,13 @@ co_WinPosShowWindow(PWND Wnd, INT Cmd)
       /* Revert focus to parent */
       if (Wnd == pti->MessageQueue->spwndFocus)
       {
+         USER_REFERENCE_ENTRY Ref;
          Parent = Wnd->spwndParent;
-         if (UserIsDesktopWindow(Wnd->spwndParent))
-             Parent = 0;
+         if (UserIsDesktopWindow(Parent))
+             Parent = NULL;
+         if (Parent) UserRefObjectCo(Parent, &Ref);
```

Comment:
Style: I don't like 1-line if(), but maybe that is just me?

Reply:
OK. Fixed.